src/Arcus.WebApi.Logging.Core/Correlation/HttpCorrelationClientOptions.cs

```diff
@@ -70,7 +78,10 @@ public string TransactionIdHeaderName
         /// <exception cref="ArgumentNullException">Thrown when the <paramref name="telemetryContext"/> is <c>null</c>.</exception>
         public void AddTelemetryContext(Dictionary<string, object> telemetryContext)
         {
-            Guard.NotNull(telemetryContext, nameof(telemetryContext), "Requires a telemetry context dictionary to add to the HTTP dependency tracking");
+            if (telemetryContext == null)
```

Comment:
```suggestion
            if (telemetryContext is null)
```
Best to use the same equalization throughout.

Reply:
This was an error on my part, good spot!

Reply:
The `is not` cannot be used in C# 8.0. So `!=` will still have to be used in at least 2 instances.

Reply:
Yes, that's fine, as long as it's used the same everywhere.